suite/tests/api/static_maps_mixup_novars.templatex

```diff
@@ -1,5 +1,6 @@
 mix up maps
 remap base=0x[0-9a-f]+, offs=0, sz=[1-9][0-9]*
 pre-DR init
-/* Depending on the environment, sometimes we hit an assert, sometimes a segfault */
```

Comment:
Why is this being removed? This does not seem related to this PR?

Reply:
This seems to be no longer the case with this change and `api.static_maps_mixup_novars_FLAKY` fails with if this string is left.

Reply:
But the static_maps_mixup app doesn't use drcachesim/drmemtrace at all.

Reply:
Plus, cpp will remove the comment, and line 5 is optional per its regex: so I don't see how removing this would make something pass that was failing.

Reply:
Ack. Changed it back to original.